modules/product/src/Tests/ProductAdminTest.php

```diff
@@ -50,11 +50,13 @@ function testAddCommerceProductAdmin() {
    * Tests creating a product with an existing SKU.
    */
   function testAddCommerceProductExistingSkuAdmin() {
-    $product = $this->createEntity('commerce_product', array(
+    $product = $this->createEntity(
+      'commerce_product', array(
       'sku' => $this->randomMachineName(),
       'title' => $this->randomMachineName(),
       'type' => 'product'
-    ));
+    )
```

Comment:
This should still be 1 line. We can switch to using [ ] to define the array, making it clearer.
(The plan was to do that across the codebase soon anyway)